### PR TITLE
update storybook with new material

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

**Review by kruai — DISMISSED** [review]
Thanks for updating all this! This is a ton of changes.

File: src/docs/api/chrome/DOC.md [hunk 1]
```diff
@@ -8,41 +8,55 @@ Insights Chrome comes with a Javacript API that allows applications to control n
...
+};
+
+/**
+ * withRouter: https://reacttraining.com/react-router/web/api/withRouter
+ * connect: https://github.com/reactjs/react-redux/blob/master/docs/api.md
+ *          https://reactjs.org/docs/higher-order-components.html
+ */
+export default withRouter (connect()(App));
+
+function buildNavigation () {
```
Comment (karelhala) [inline src/docs/api/chrome/DOC.md]: This is no longer needed, am I right?

**Review by karelhala — DISMISSED** [review]
Looking good, just one question about app usage.

File: src/docs/api/chrome/DOC.md (hunk 1 above)
Comment (ryelo) [reply src/docs/api/chrome/DOC.md]: I don't think it is. I just copy-pasted from insights-chrome readme

**Review by karelhala — APPROVED** [review]
Looking good! Thanks for these updates.

**Pull request merged**